tensorflow/core/kernels/mkl/mkl_conv_ops.cc

```diff
@@ -987,6 +982,13 @@ class MklConvOp : public OpKernel {
             const_cast<Toutput*>(add_tensor.flat<Toutput>().data()));
         void* dst_buf =
             static_cast<void*>((*output_tensor)->flat<Ttemp_output>().data());
+        if (native_format) {
+          // We are simply deep copying the add_tensor to output_tensor without
+          // changing memory layout, hence using same memory descriptor.
+          ADD_MD = DST_MD =
```

Comment:
Nit: I think we can just use `add_md` and `dst_md` here since the macros for v0.x has ready been deleted anyway. But if you'd rather keep `ADD_MD` and `DST_MD` to be consistent (for a later clean-up), I'm fine with that too.
```suggestion
          add_md = dst_md =
```

Reply:
Let's keep it like that since my colleague is preparing another PR to cleanup the whole file.

Reply:
Sounds good!